src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -1265,6 +1284,39 @@ class ApplicationManagerImpl
    */
   void SendMobileMessage(smart_objects::SmartObjectSPtr message);
 
+  /*
+   * @brief Sets default value of the HELPPROMT global property
```

Comment:
@mked-luxoft why uppercase?

Reply:
@LuxoftAKutsan i suppose it is the name of a constant representing global property id